docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -8,9 +8,13 @@ FROM ubuntu:${ubuntu_version} AS common_base
 # Ubuntu Setup
 ENV TZ=America/Chicago
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+ENV HOME /root
```

Comment:
Docker generally comes with root user privilege. I think it is not necessary to use extra env.

Reply:
I think it's still necessary to specify the ENV variable HOME since it's used as a variable in a lot of other lines in the dockerfile.

Reply:
Let's stick with this for now since we have historically installed in `/root/opt` rather than `/opt` to have some control over what's there.